astroquery/jplspec/lookup_table.py

```diff
@@ -22,10 +22,13 @@ def find(self, s, flags):
 
         Returns
         -------
-        The list of values corresponding to the matches
+        The dictionary containing only values whose keys match the regex
 
         """
 
+        if s in self:
```

Comment:
ahh, my eyes hurt, could we make this called something longer than s? I suspect that would mean a deprecation--rename?

Reply:
we renamed it to `st` in the other instance.  But this particular argument is _barely_ user-facing; I don't know that we need to deprecate it